doc/developers/contributing.rst

```diff
@@ -40,6 +40,10 @@ ticket to the
 `GitHub issue tracker <https://github.com/scikit-learn/scikit-learn/issues>`_. You are
 also welcome to post feature requests or pull requests.
 
+Governance
+----------
+The decision making process and governance structure of scikit-learn is laid
+out in the governance document :ref:`governance`.
```

Comment:
Please insert a colon before the `:ref:` marker:

... out in the governance document: :ref:`governance`.

The `:ref:` marker is replaced by the title of the referenced section in the rendered HTML:

https://54937-843222-gh.circle-artifacts.com/0/doc/developers/contributing.html#governance